src/plugins/kibana_react/public/index.ts

```diff
@@ -47,6 +47,7 @@ export {
 export { useExecutionContext } from './use_execution_context';
 
 export type { ToolbarButtonProps } from './toolbar_button';
+/** @deprecated ToolbarButton - use `ToolbarButton` from `@kbn/shared-ux-button-toolbar` */
```

Comment:
I assume this was missed in the original deprecation? This is the `ToolbarButton` that Lens is currently using, so I figured it might be worth deprecating as well... Didn't want to muddle up this PR even more by going in to Lens code to migrate them over to the Shared UX toolbar though, so I figured deprecating was a good enough first step 😅 

Let me know if deprecating this was not the right call, though.